october-2017/17.md

```diff
@@ -60,4 +60,4 @@ All that we know for now is that the MyBB 1.x line needs to be improved, and we
 ### Moving Forward
 Overall we think these changes will help bring MyBB into the modern age, and make things a lot more fun to take part in. We love reading what you, the users of MyBB have to say and take it *all* into consideration. If you'd like to have a discussion with us, please hop on Discord and let us know about it!
 
-[![Join Discord](../resources/discord.svg)](https://discordapp.com/invite/rX8VpBr)
\ No newline at end of file
+[![Join Discord](../resources/discord.svg)](https://discordapp.com/invite/rX8VpBr)
```

Comment:
Not seeing any diff here, make sure to check the newlines. Otherwise looks good.